test/io/jade/readBits-castchecking.chpl

```diff
@@ -0,0 +1,56 @@
+use FileSystem;
+use IO;
+
+config const testfile="test.bin";
+var f = open(testfile, ioMode.cwr);
+
+var w = f.writer(kind=ionative);
+var b = [0xaa66, 0x7852, 0x9287, 0x1455, 0x2299, 0x7218, 0x8278, 0x4082] :uint(16);
```

Comment:
This won't be portable between big endian and little endian systems. But it will be if you make this an array of bytes instead, so I recommend that.